Tests/Sentry.Xamarin.Tests/Sentry.Xamarin.Tests.csproj

```diff
@@ -1,27 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
-
+  <TargetFrameworks>net5.0;netcoreapp3.1;netcoreapp3.0</TargetFrameworks>
```

Comment:
Curious why test Xamarin code under 3 versions of .NET Core?

Reply:
This specific code is not dependent on Xamarin, I made it that way for having some tests that would be able to run with Github Actions (PR welcome for adding the test action for this csproj :D )